### PR TITLE
feat: aura vaults promotion script

### PR DESCRIPTION
**Pull request by sajanrajdev**
Tackles #732 

Promotes the following vaults to guarded state:
```
    "bauraBal",
    "b80BADGER_20WBTC",
    "b40WBTC_40DIGG_20graviAURA",
    "bBB_a_USD",
    "b33auraBAL_33graviAURA_33WETH"
```

Run with:
```
brownie run scripts/issue/732/promote_aura_vaults.py
```

### CODE REVIEW

**Comment by sajanrajdev** [comment]
Call and events trace:
```
Running 'scripts/issue/732/promote_aura_vaults.py::main'...
Transaction sent: 0x6dbf5f619e40236e570cc8ab0b7c06d313432239b7abcc45f6eaceba836ba7b1
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 1
  IBadgerRegistryV2.promote confirmed   Block: 15316829   Gas used: 57392 (0.48%)

Promote: 0x37d9D2C6035b744849C15F1BFEE8F268a20fCBd8 (v1.5) to 1
Transaction sent: 0x94801ffa4da9bfac69cc511b44d0516d36a774352934481a8cf242d42a7df58b
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 2
  IBadgerRegistryV2.promote confirmed   Block: 15316830   Gas used: 58152 (0.48%)

Promote: 0x63ad745506BD6a3E57F764409A47ed004BEc40b1 (v1.5) to 1
Transaction sent: 0x911d405f77084b2522e4a5cd6366e9d003a7e43c5ab754b3233c88e69949c306
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 3
  IBadgerRegistryV2.promote confirmed   Block: 15316831   Gas used: 59996 (0.50%)

Promote: 0x371B7C451858bd88eAf392B383Df8bd7B8955d5a (v1.5) to 1
Transaction sent: 0xf1847dfe0325b736c0159bbb4acbea5a988ebc71878a2ac6ba2328de89b84d0b
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 4
  IBadgerRegistryV2.promote confirmed   Block: 15316832   Gas used: 57477 (0.48%)

Promote: 0x06D756861De0724FAd5B5636124e0f252d3C1404 (v1.5) to 1
Transaction sent: 0x4a87953592d1f2ae21f7999a0e5826395ce85083c91016f45cb1a3b8dc23bc21
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 5
  IBadgerRegistryV2.promote confirmed   Block: 15316833   Gas used: 60683 (0.51%)

Promote: 0xf8f5677B6bCecdb9be94AE8f6770a05a6C53C378 (v1.5) to 1
Transaction sent: 0xacb5486648f5a33eea4d7afc4ffe8d7bdaeff6af2a80cb7125ec738465146242
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 27
  Gnosis Safe.approveHash confirmed   Block: 15316829   Gas used: 46206 (0.39%)

Transaction sent: 0x81f22592baf609b209588763115b52d0d1d133d99276903fe620ee489915181a
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 18
  Gnosis Safe.approveHash confirmed   Block: 15316830   Gas used: 46206 (0.39%)

Transaction sent: 0x9f2c9d5e7eb0134178b0f77c8c980a3407b43e794722faa8d44019feaa0705f2
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 10
  Gnosis Safe.approveHash confirmed   Block: 15316831   Gas used: 46206 (0.39%)

Transaction sent: 0x577b41ca8e286f9e2a9c00c2dcbbc98a0538cd3cd335f13a1728ac33270a6bdd
  Gas price: 0.0 gwei   Gas limit: 270092   Nonce: 28
  Gnosis Safe.execTransaction confirmed   Block: 15316832   Gas used: 226254 (83.77%)

Transaction was Mined
---------------------
Tx Hash: 0x577b41ca8e286f9e2a9c00c2dcbbc98a0538cd3cd335f13a1728ac33270a6bdd
From: 0x0a9af7FAba0d5DF7A8C881e1B9cd679ee07Af8A2
To: 0x86cbD0ce0c087b482782c181dA8d191De18C8275
Value: 0
Function: Gnosis Safe.execTransaction
Block: 15316832
Gas Used: 226254 / 270092 (83.8%)

Events In This Transaction
--------------------------
└── Gnosis Safe (0x86cbD0ce0c087b482782c181dA8d191De18C8275)
    └── ExecutionSuccess
        ├── txHash: 0x5261579fdcd476de1ddbb480a15e6158dccdeda388f8aad715e952b7318dd76c
        └── payment: 0

```

File: scripts/issue/732/promote_aura_vaults.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+from helpers.addresses import registry
+from rich.console import Console
+
+C = Console()
+
+TECH_OPS = registry.eth.badger_wallets.techops_multisig
+DEV = registry.eth.badger_wallets.dev_multisig
+REGISTRY = registry.eth.registry_v2
+VAULTS = registry.eth.sett_vaults
+STATUS = 1 # Guarded
```
Comment (axejintao) [inline scripts/issue/732/promote_aura_vaults.py:12]:
```
  enum VaultStatus {
    deprecated,
    experimental,
    guarded,
    open
  }
  ```

this should be status: 2 

i believe ser

**Review by axejintao — CHANGES_REQUESTED** [review]

File: scripts/issue/732/promote_aura_vaults.py (hunk 1 above)
Comment (sajanrajdev) [reply scripts/issue/732/promote_aura_vaults.py:12]: Good catch, you are right. Changing..

**Review by axejintao — APPROVED** [review]

**Review by petrovska-petro — APPROVED** [review]
Promotion to guarded state of the targeted vaults looks correct ✅

**Review by shuklaayush — APPROVED** [review]

**Review by dapp-whisperer — APPROVED** [review]

**Pull request merged**